app/views/docs/installation.phtml

```diff
@@ -56,6 +56,14 @@
     </li>
 </ul>
 
+<p>You will be propted to configure the following during setup:</p>
```

Comment:
What do you think about moving this to before the Unix `<h3>`? The structure/nesting is a little odd because it makes it seem like this setup is only for the Windows `<h3>`.

Reply:
<img width="908" alt="Screenshot 2023-01-25 at 10 32 14 PM" src="https://user-images.githubusercontent.com/29069505/214753271-4aefe5e9-fc92-40d8-935c-829238af7c71.png">